wormhole-connect/src/components/Background/BackgroundImage.tsx

```diff
@@ -118,21 +117,8 @@ type Props = {
 function Background({ children }: Props) {
   const { classes } = useStyles();
 
-  return THEME.background.default === 'wormhole' ? (
-    <div className="container">
-      <div className={classes.bg}>
-        {children}
-        <div className={classes.circles}></div>
-        <div className={classes.background}></div>
-        <div className={classes.gradientRight}></div>
-        <div className={classes.gradientRight2}></div>
-        <div className={classes.gradientLeft}></div>
-        <div className={classes.gradientLeft2}></div>
-      </div>
-    </div>
-  ) : (
-    <div className={classes.children}>{children}</div>
-  );
+  // TODO remove this component
```

Comment:
Should we remove the `Background` component?

Reply:
yeah thanks :) done